### PR TITLE
feat: susde deployment

### PR DESCRIPTION
**Pull request by ioanaroceanu2000**

### CODE REVIEW

File: packages/tomls/src/token/tokens/susde.toml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+[var.susde_vars]
+susdeTokenName = "Ethena sUSDe"
+susdeTokenSymbol = "sUSDE"
```
Comment (arturbeg) [inline packages/tomls/src/token/tokens/susde.toml:3]: small point, but can we double check and make sure the symbol and token name match what you'd see on other networks?

File: packages/tomls/src/token/tokens/susde.toml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    [
+        "<%= settings.pauser1 %>",
+        "<%= settings.pauser2 %>",
+        "<%= settings.pauser3 %>",
+        "<%= settings.pauser4 %>",
+    ],
+]
+depends = ["invoke.upgrade_susde_proxy"]
+
+# [invoke.susde_mint_permission]
```
Comment (arturbeg) [inline packages/tomls/src/token/tokens/susde.toml:58]: will invoke these once socket is done with their deployments right?

**Review by arturbeg — DISMISSED** [review]

File: packages/tomls/src/token/tokens/susde.toml (hunk 2 above)
Comment (ioanaroceanu2000) [reply packages/tomls/src/token/tokens/susde.toml:58]: correct

File: packages/tomls/src/token/tokens/susde.toml (hunk 1 above)
Comment (ioanaroceanu2000) [reply packages/tomls/src/token/tokens/susde.toml]: [1f88029](https://github.com/Reya-Labs/reya-deployments/pull/101/commits/1f8802955b18203ba9032499f04afcdf404c4878)

**Review by arturbeg — APPROVED** [review]

**Pull request merged**